src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -40,6 +41,9 @@
          * to think about number formatting issues.
          */
         Utils.forceJVMLocaleToUSEnglish();
+
+        // Set config factory to know about the configuration options that we have.
```

Comment:
Is this static block working for extending classes? If another static block is added with different code (also config-related, but I've just realized that the forcing locale may be the same), will this be initialized? And if there is another initialization of the config, will this happen before or after?

I think that all this should be noted in the javadoc for the class. Maybe this can be done in a `Main` constructor instead?